packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/Editor.tsx

```diff
@@ -191,7 +191,6 @@ function Editor(props: EditorProps, ref: any) {
 		cm.on('drop', editor_drop);
 		cm.on('dragover', editor_drag);
 		cm.on('refresh', props.onResize);
-		cm.on('update', props.onResize);
```

Comment:
Thanks for looking into it @ken1kob. The fact that we no longer call "onResize" in this case, doesn't it mean that the editor will no longer be properly updated in some cases, for example when it's resized?

Reply:
Since resize events are covered by CodeMirror's 'refresh' events, they are not in CodeMirror's 'update' events.

I have not yet found specific event in CodeMirror's update events needed to be ignored. They were ignored on the assumption that scroll events after update events are not originated by user. The found counter-examples (such as local search) shows the assumption is wrong.